Makefile

```diff
@@ -18,8 +26,28 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
+CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
+
+ifeq ($(TARGET_S),android)
```

Comment:
Is it possible for android to target x86_64? If so ca we please only execute this block if the target arch is aarm64? If not, can we please warn & error if the user provides a non existant OS & ARCH?

Reply:
Hm.. good question. I guess it is possible although unlikely to have an android/x86_64 target.

Added error case here if android and not arm64.